Logger/BaseLogger.cs

```diff
@@ -2,6 +2,8 @@
 {
     public abstract class BaseLogger
     {
+        public string ClassName { get; set; } = "";
```

Comment:
```suggestion
        public string ClassName { get; set; }
```
No need to set this equal to an empty string.